From 6cd7c89c36d0f35af3e728241ac1ac2265d65b85 Mon Sep 17 00:00:00 2001
From: Hugo Osvaldo Barrera <hugo@whynothugo.nl>
Date: Sun, 20 Sep 2026 22:44:33 +0200
Subject: [PATCH] Fix undefined type, uint

This file fails to build on musl with:

	tests/test_utils_parsers.cpp:26:23: error: 'uint' was not declared in this scope; did you mean 'int'?

Explicitly declare uint for this test.
---
 tests/test_utils_parsers.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tests/test_utils_parsers.cpp b/tests/test_utils_parsers.cpp
index 91fa1660..9f50deb8 100644
--- a/tests/test_utils_parsers.cpp
+++ b/tests/test_utils_parsers.cpp
@@ -7,6 +7,7 @@
 #include "Utils/Parsers.h"
 
 using namespace gamescope;
+using uint = unsigned int;
 
 TEST_CASE("Utils/Parsers", "[parsers]") {
     SECTION("int") {
-- 
2.55.0

